### ISSUE TITLE
Add alt text to figures in reports

### ISSUE THREAD

**Issue by Bisaloo** [european-modelling-hubs/covid19-forecast-hub-europe_archive#1064]
This is especially useful for visitors using screen readers.

It may also be a good moment to switch to [knitr's new syntax for chunks](https://github.com/yihui/knitr/releases/tag/v1.35). Another nice side effect is that diffs in git will be clearer.

I need to ask what is the best practice to have a meaningful alt text in the case of continuously updated figures. We cannot really describe what is going on since we don't know it yet.
